### PR TITLE
Third

### PR DESCRIPTION
**Pull request by mariosknl**
Project: embedded images and video.

replicating a youtube page as per pathwright requirements.

using HTML  and CSS

### CODE REVIEW

**Comment by shshamim63** [comment]
## Status :red_circle:  [Changes Required] :red_circle: 
___
Hi @mariosknl  and @kotoku-code :wave: ,
Awesome initiative :clap: . Project contains all the items that are needed to decorate the page. Contains a readme. But some issues are blocking to approve this project.
___
#### Please follow the instructions below to complete this project.
___
### :spades: [Mandatory Changes] :spades: 
___
1. You can decorate the readme with some styling. Checkout this [link](https://www.markdownguide.org/cheat-sheet/)
2. As you have configured the stickler on your repository. lets complete it and fix the errors. So to do so, please follow the instructions mentioned in this [link](https://github.com/microverseinc/linters-config/tree/master/css).
3. You were asked to check HTML markup with this validator https://validator.w3.org/#validate_by_upload. I did so and saw an error.
image
4. Why there is a >DS_store file. Files not related to project should not be in there. So please remove it.
___
#### Please fix the above mentioned issue and send another review request.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawathossainshamim/)

**Comment by kotoku-code** [comment]
we have already corrected these issues but you are seeing an older version. how can we fix that

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (adriaanbd) [inline .gitignore]: Please add `.DS_Store` to this file:

```
node_modules
.DS_Store
```

After doing that, you need to remove the file from this repository. It shouldn't be committed to the repo. Right now it's here:

```
$ l -a
./   .DS_Store  .gitignore  normalize.css  README.md
../  .git/      index.html  pictures/      style.css
```

[`REQUIRED`]

File: index.html [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+    </nav>
+    <main>
+      <embed
+        width="560"
+        height="315"
+        src="https://www.youtube.com/embed/V74l_zS1x8E"
+        frameborder="0"
+        allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+        allowfullscreen
+      ></embed>
```
Comment (adriaanbd) [inline index.html]: This line is giving an error in the validator:
image

Please fix [`REQUIRED`]

File: README.md [hunk 3]
```diff
@@ -1,2 +1,29 @@
...
+Built With
+
+HTML & CSS
+
+Live Demo Link
+
+https://raw.githack.com/mariosknl/Embedding-Images-and-Video/third/index.html
+
+Authors
+👤 Author1
```
Comment (adriaanbd) [inline README.md]: Add a space between Authors and Author1 so that Authors1 is in a new line [`OPTIONAL`]

File: README.md [hunk 4]
```diff
@@ -1,2 +1,29 @@
...
+
+Marios Kanellopoulos
+Github: https://github.com/mariosknl
+Twitter: @MariosKnl
+Linkedln: https://www.linkedin.com/in/marios-kanellopoulos-a99332181/
+Φ
+
+Author 2
```
Comment (adriaanbd) [inline README.md]: Don't you want the same icon to the left of this as the one above? [`OPTIONAL`]

File: README.md [hunk 5]
```diff
@@ -1,2 +1,29 @@
...
+
+kotoku kennedy
```
Comment (adriaanbd) [inline README.md]: Lowercase name or Title Case name? [`OPTIONAL`]

**Comment by mariosknl** [comment]
all corrections made!please review again!

File: index.html [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+        <li>
+          <a href="#"><i class="fas fa-th"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fas fa-bell"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fas fa-user-circle"></i></a>
+        </li>
+      </ul>
```
Comment (suretrust) [inline index.html]: Use embedded images and not icons. [Requirement from pathright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/icons_requirement.png)

Comment (kotoku-code) [reply index.html]: we have embedded 3 images as per the requirement from pathright

**Pull request merged**